### ISSUE TITLE
Error in Calculating AUROC?

### ISSUE THREAD

**Issue by jqtangust** [vijaylee/Continual_Anomaly_Detection#1]
To average all objects to get mean AUROC in each step.

**Comment by vijaylee** [comment]
I don't quite understand your question.

**Comment by vijaylee** [comment]
I don't quite understand your question. Could you provide more specific details about your issue?

**Comment by jqtangust** [comment]
In current code, the average AUROC is calculated by Step B: (Task A + Task B) / 2. However, Task A may contain 10 objects, Task B may contain 1 objects, so to get mean AUROC, you may need to do like:  Step B:  (Task A * 10 + Task B * 1) / 11.

**Comment by vijaylee** [comment]
You are right, thanks for pointing out the error, we have modified the code.
